Ports/faad2/package.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=faad2
+version=2_10_0
+useconfigure=true
+files="https://github.com/knik0/faad2/archive/refs/tags/${version}.tar.gz faad2-${version}.tar.gz"
+
+post_fetch() {
+    cd "faad2-${version}"
+    autoreconf -vif
+    cd ..
```

Comment:
That's exactly what `run` does :)

https://github.com/SerenityOS/serenity/blob/15a820e9ad47fa61b6010fac8dbd58c6ea853f81/Ports/.port_include.sh#L41-L44

```suggestion
    run autoreconf -vif
```

Reply:
Did not know this! Will fix